src/client/features/pathways/index.js

```diff
@@ -84,9 +86,32 @@ class Pathways extends React.Component {
     }
   }
 
+  baseNameToUrl(baseName){
+    const lowercaseName = baseName.toLowerCase();
+
+    let databaseURL = databaseURLMap.get(lowercaseName);
+    if(!databaseURL){
+      const datasourceURLScores = [];
+      databaseURLMap.forEach( (datasourceURL, datasource, map) => {
+        datasourceURLScores.push([datasourceURL,diceCoefficient(datasource,lowercaseName)]);
+      });
+      datasourceURLScores.sort( (a,b) => { return b[1] - a[1]; } );
+      databaseURL = datasourceURLScores[0][0];
+    }
+
+    return databaseURL;
+  }
+
   render() {
     let { loading, pathway, cySrv, activeMenu, networkEmpty } = this.state;
 
+    const baseDatasource = pathway.datasource();
+    const databaseURL = this.baseNameToUrl(baseDatasource);
+    let baseName = pathway.name();
+    if(baseName.toLowerCase() === 'unknown title'){
```

Comment:
Maybe you can just change the default value returned by pathway.name().  What do you think?

Reply:
I can't figure out where `pathway.name()` is being defined, do you know?

Reply:
https://github.com/PathwayCommons/app-ui/tree/development/src/client/models/pathway